src/test/java/uk/gov/hmcts/reform/rsecheck/errorhandling/GlobalExceptionHandlerTest.java

```diff
@@ -23,12 +24,23 @@ public void testHandleSubscriptionNotFoundMethod() {
         PublicationNotFoundException publicationNotFoundException
             = new PublicationNotFoundException("This is a test message");
 
+        CourtNotFoundException courtNotFoundException
+            = new CourtNotFoundException("This is a test message for court");
+
         ResponseEntity<ExceptionResponse> responseEntity =
             globalExceptionHandler.handlePublicationNotFound(publicationNotFoundException);
 
         assertEquals(HttpStatus.NOT_FOUND, responseEntity.getStatusCode(), "Status code should be not found");
         assertNotNull(responseEntity.getBody(), "Response should contain a body");
         assertEquals("This is a test message", responseEntity.getBody().getMessage(),
                      "The message should match the message passed in");
+
```

Comment:
your tests wont pass, make sure you try tun them before pushing

Reply:
actually the whole project doesnt build currently becuase of this

Reply:
test corrected